lib/samples/add_map_image_layer/add_map_image_layer_sample.dart

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+
+class AddMapImageLayerSample extends StatefulWidget {
+  const AddMapImageLayerSample({super.key});
+
+  @override
+  State<AddMapImageLayerSample> createState() => _AddMapImageLayerSampleState();
+}
+
+class _AddMapImageLayerSampleState extends State<AddMapImageLayerSample>
+    with SampleStateSupport {
+  // Create a map view controller.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready.
+  var _ready = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Because of how simple this sample is, we really only need the Scaffold and the ArcGISMapView. We don't need the SafeArea, the Stack, or the _ready overlay thingy because there are no controls or tap handler. So those can all get deleted. And then there's no need for the _ready variable.